napari_allencell_annotator/view/annotator_view.py

```diff
@@ -309,16 +321,19 @@ def _create_annot(self, name: str, dictn: Dict):
             item.setValue(dictn["default"])
         elif annot_type == "bool":
             item = QCheckBox()
-            if dictn["default"] == "true" or dictn["default"]:
+            if dictn["default"] == "true":
```

Comment:
You should never have to do this- it means you loaded in the json wrong [here](https://github.com/aics-int/napari-allencell-annotator/blob/78b2e7becdb0eed80ea86b0c66248710f5e49d2b/napari_allencell_annotator/controller/annotator_controller.py#L177)

instead of `json.load()` do this to convert the json into python objects and variables:

```
    with open("jsonfilepath", "r") as f:
        unpacked = json.loads(f.read())
```